apps/hash-frontend/src/pages/shared/not-archived-filter.ts

```diff
@@ -0,0 +1,29 @@
+import { Filter } from "@local/hash-graph-client";
+import { systemTypes } from "@local/hash-isomorphic-utils/ontology-types";
+import { extractBaseUrl } from "@local/hash-subgraph/type-system-patch";
+
+const archivedBaseUrl = extractBaseUrl(
+  systemTypes.propertyType.archived.propertyTypeId,
+);
+
+export const notArchivedFilter: Filter = {
+  any: [
+    {
+      equal: [
+        {
+          path: ["properties", archivedBaseUrl],
+        },
+        // @ts-expect-error -- We need to update the type definition of `EntityStructuralQuery` to allow for this
```

Comment:
Thanks for adding this! Maybe let's link the task we created the other day here:
```suggestion
        // @ts-expect-error -- We need to update the type definition of `EntityStructuralQuery` to allow for this
        //   @see https://linear.app/hash/issue/H-1207
```

We could also make use of this `notArchivedFilter` in additional places in the codebase, including:
- in `notifications-context.tsx`
- in `notification.tsx`
- I believe also in the BE once you merge in the latest changes from `main`, but that'd require moving this into a shared package

Reply:
Thank you, done in https://github.com/hashintel/hash/pull/3540/commits/84fd8f8b6271d19aad7ef99459d39e8553ac02b3